### PR TITLE
Content about prototyping high risk items first in principles…

### PR DESCRIPTION
**Pull request by Yemaneberhan-Lemma**
Added content about "prototyping high risk items first" inside principles -> fundamentals page.

### CODE REVIEW

**Review by hussainweb — CHANGES_REQUESTED** [review]
@Yemaneberhan-Lemma, why do you think "Focus on Fundamentals" is the right place for this text? Especially, why is it under the section of "Working with source code"?

Looking at the other principles, maybe this is a better fit under "Focus on Engineering", IMHO. Or what do you think of introducing a new principle related to "Iterations"? cc @axelabhay

**Comment by axelabhay** [comment]
> @Yemaneberhan-Lemma, why do you think "Focus on Fundamentals" is the right place for this text? Especially, why is it under the section of "Working with source code"?
> 
> Looking at the other principles, maybe this is a better fit under "Focus on Engineering", IMHO. Or what do you think of introducing a new principle related to "Iterations"? cc @axelabhay


 > Why do you think "Focus on Fundamentals"  is the right place for this text?

I think, suggestion at https://docs.google.com/document/d/1gBTKAwxjZvxTeyPtemEQbrRo5dyBhQji-tOGzw_vyXE/edit?disco=AAAA4YCsPrs has been followed. :)
 
> Especially, why is it under the section of "Working with source code"?

+1, it is `H3 heading` and falling under the section "Working with source code".

> Looking at the other principles, maybe this is a better fit under "Focus on Engineering", IMHO. Or what do you think of introducing a new principle related to "Iterations"?

The first line of this content talks about `architectural obstacles` and I think architecting is an iterative process. Yes, we can introduce "Iterations".

**Comment by hussainweb** [comment]
Thanks, @axelabhay. I see I caused the confusion. Reading it now on the "Focus on Fundamentals" page seems a little weird to me. Maybe I didn't realize it earlier or I had something else in mind and have forgotten it now.

@Yemaneberhan-Lemma I think this is more related to Engineering and it might flow there. Or maybe a new page under Principles is a better choice. We can make Iterations a principle in itself and put this under there.

**Comment by Yemaneberhan-Lemma** [comment]
> Thanks, @axelabhay. I see I caused the confusion. Reading it now on the "Focus on Fundamentals" page seems a little weird to me. Maybe I didn't realize it earlier or I had something else in mind and have forgotten it now.
> 
> @Yemaneberhan-Lemma I think this is more related to Engineering and it might flow there. Or maybe a new page under Principles is a better choice. We can make Iterations a principle in itself and put this under there.

@hussainweb: New page is ideal for me. And noted on creating new content/page for Iterations.

File: content/docs/principles/prototyping-high-risk-items-first.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Prototyping high risk items first
```
Comment (hussainweb) [inline content/docs/principles/prototyping-high-risk-items-first.md:2]: Let's call the page "Build in Iterations". Please change the file name too.

File: content/docs/principles/prototyping-high-risk-items-first.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+weight: 1
```
Comment (hussainweb) [inline content/docs/principles/prototyping-high-risk-items-first.md:3]: I don't think this should be the first. Let's keep this one after "Focus on Fundamentals".

**Review by hussainweb — APPROVED** [review]
Nevermind, I'll merge for now and rename later during my review.

**Pull request merged**